src/Network/Broadcast/Relay/Class.hs

```diff
@@ -29,7 +29,7 @@ data Relay packingType m where
       , Msg.Message (ReqMsg key)
       , Msg.Message (InvOrData key contents)
       )
-      => (PropagationMsg packingType -> m ()) -- ^ How to relay the data.
```

Comment:
Yes, Haddock was broken because of these lines. I don't know the reason, it just didn't parse.